src-tauri/src/database.rs

```diff
@@ -140,36 +133,24 @@ pub fn find_all_game_files(game_id: &str) -> Vec<String> {
   files
 }
 
-pub fn find_game_by_id(id: &str) -> Option<Game> {
-  let game = load_game_with_meta(id);
-  return Some(game);
+pub fn find_game_by_id(id: &str) -> Option<DbGameMeta> {
+  let db_game = load_game_with_meta(id);
+  return Some(db_game);
 }
 
-pub fn save_game(game: Game) {
-  let json_meta_path = get_meta_directory().join(&game.name).join("meta.json");
-
-  let game_meta_json = DbGameMeta {
-    rating: Some(game.rating),
-    description: Some(game.description),
-    notes: Some(game.notes),
-    tags: Some(game.tags),
+pub fn save_game(db_game: DbGameMeta) {
```

Comment:
This function was the real driver for using the "Db" variants: it needed to add `previous_file_state: game.previous_file_state`, but since that's a function that requires the graphql context instead of a simple property, things got messy.

I tried changing the signature to `save_game(game: Game, previous_file_state: Vec<PreviousFileState>)`, but that had too many drive-by changes for my liking.

I think it ends up being a little more clean. The `database` module really now only cares about database related models, but the GraphQL layer has to worry about both sides a lot more often.